JobConfig/primary/flatInternalRPC.fcl

```diff
@@ -1,35 +1,18 @@
 #
 # internal RPC with a flat virtual photon energy spectrum
 #
-#include "JobConfig/primary/prolog.fcl"
-process_name: flatInternalRPC
-
-source: { module_type : EmptyEvent }
-services: @local::Services.Sim
-physics : { 
-  producers : {
-    generate : @local::EventGenerator.producers.RPCGun
-    @table::Primary.producers
-  }
-  filters: { @table::Primary.filters }
-  analyzers: { @table::Primary.analyzers }
-  TriggerPath : @local::Primary.TriggerPath
-  EndPath : @local::Primary.EndPath
-}
-outputs : { 
-  Output :@local::Primary.Output
-}
-#
-# final configuration
-#
-#include "JobConfig/primary/epilog.fcl"
+#include "JobConfig/primary/primary.fcl"
+# set the generator
+physics.producers.generate : @local::EventGenerator.producers.RPCGun
+# set the stops.  This should go away with art-based muon stops FIXME!
+physics.producers.generate.muonStops.inputFiles : [ "mergedMuonStops/nts.mu2e.DS-TGTstops.MDC2018a.001002_00000000.root" ]
```

Comment:
These are the only official stops from MDC2018 (we never processed stopped pions).  It is temporary, for testing, it will go away when we have art-based stops.

Reply:
I am reluctant to have something explicitly wrong set to run out of the box.   It would be better to set inputFile to @nil here.   We can keep the mergedMuonStops filename in a comment nearby to make it more convenient to test.

Reply:
OK, I replaced with @nil